Failed to write backup file: Automatic backup fails due to the timeout

We have a fairly large number of comments (~160Mb uncompressed) 
therefore an automatic backup fails after 15 minutes due to the timeout:

```
$ remark42 backup -s remark
remark42 v1.9.0-0521c702-20220124T12:35:27
2022/03/15 04:29:07.000 [INFO]  export to ./backup/, site remark
2022/03/15 04:44:07.044 [WARN]  failed to close response, context deadline exceeded
2022/03/15 04:44:07.044 [ERROR] failed with context deadline exceeded
failed to write backup file backup/userbackup-remark-20220315T042907.gz
github.com/umputun/remark42/backend/app/cmd.(*BackupCommand).Execute
	/build/backend/app/cmd/backup.go:76
main.main.func1
	/build/backend/app/main.go:55
github.com/umputun/go-flags.(*Parser).ParseArgs
	/build/backend/vendor/github.com/umputun/go-flags/parser.go:331
github.com/umputun/go-flags.(*Parser).Parse
	/build/backend/vendor/github.com/umputun/go-flags/parser.go:190
main.main
	/build/backend/app/main.go:62
runtime.main
	/usr/local/go/src/runtime/proc.go:255
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1581
2022/03/15 04:44:07.044 [ERROR] failed with context deadline exceeded
failed to write backup file backup/userbackup-remark-20220315T042907.gz
github.com/umputun/remark42/backend/app/cmd.(*BackupCommand).Execute
	/build/backend/app/cmd/backup.go:76
main.main.func1
	/build/backend/app/main.go:55
github.com/umputun/go-flags.(*Parser).ParseArgs
	/build/backend/vendor/github.com/umputun/go-flags/parser.go:331
github.com/umputun/go-flags.(*Parser).Parse
	/build/backend/vendor/github.com/umputun/go-flags/parser.go:190
main.main
	/build/backend/app/main.go:62
runtime.main
	/usr/local/go/src/runtime/proc.go:255
runtime.goexit
	/usr/local/go/src/runtime/asm_amd64.s:1581
failed to write backup file backup/userbackup-remark-20220315T042907.gz: context deadline exceeded
```

**Is there are a reason for having it set to 15 minutes by default? 
Would that make sense to increase the default `timeout` value to `60m`?**

https://github.com/umputun/remark42/blob/e7950fcb22cca618f90db0db2218741f3847acab/backend/app/cmd/backup.go#L21

```
// BackupCommand set of flags and command for export
// ExportPath used as a separate element to leverage BACKUP_PATH. If ExportFile has a path (i.e. with /) BACKUP_PATH ignored.
type BackupCommand struct {
	ExportPath  string        `short:"p" long:"path" env:"BACKUP_PATH" default:"./var/backup" description:"export path"`
	ExportFile  string        `short:"f" long:"file" default:"userbackup-{{.SITE}}-{{.TS}}.gz" description:"file name"`
	Site        string        `short:"s" long:"site" env:"SITE" default:"remark" description:"site name"`
	Timeout     time.Duration `long:"timeout" default:"15m" description:"export (backup) timeout"`
	AdminPasswd string        `long:"admin-passwd" env:"ADMIN_PASSWD" required:"true" description:"admin basic auth password"`
	CommonOpts
}
```

--- a/backend/app/cmd/backup.go
+++ b/backend/app/cmd/backup.go
@@ -18,7 +18,7 @@ type BackupCommand struct {
 	ExportPath  string        `short:"p" long:"path" env:"BACKUP_PATH" default:"./var/backup" description:"export path"`
 	ExportFile  string        `short:"f" long:"file" default:"userbackup-{{.SITE}}-{{.TS}}.gz" description:"file name"`
 	Site        string        `short:"s" long:"site" env:"SITE" default:"remark" description:"site name"`
-	Timeout     time.Duration `long:"timeout" default:"15m" description:"export (backup) timeout"`
+	Timeout     time.Duration `long:"timeout" default:"60m" description:"export (backup) timeout"`
 	AdminPasswd string        `long:"admin-passwd" env:"ADMIN_PASSWD" required:"true" description:"admin basic auth password"`
 	CommonOpts
 }

--- a/backend/app/cmd/import.go
+++ b/backend/app/cmd/import.go
@@ -19,7 +19,7 @@ type ImportCommand struct {
 	InputFile   string        `short:"f" long:"file" description:"input file name" required:"true"`
 	Provider    string        `short:"p" long:"provider" default:"disqus" choice:"disqus" choice:"wordpress" choice:"commento" description:"import format"` //nolint
 	Site        string        `short:"s" long:"site" env:"SITE" default:"remark" description:"site name"`
-	Timeout     time.Duration `long:"timeout" default:"15m" description:"import timeout"`
+	Timeout     time.Duration `long:"timeout" default:"60m" description:"import timeout"`
 	AdminPasswd string        `long:"admin-passwd" env:"ADMIN_PASSWD" required:"true" description:"admin basic auth password"`
 	CommonOpts
 }

--- a/backend/app/cmd/remap.go
+++ b/backend/app/cmd/remap.go
@@ -18,7 +18,7 @@ type RemapCommand struct {
 	Site        string        `short:"s" long:"site" env:"SITE" default:"remark" description:"site name"`
 	InputFile   string        `short:"f" long:"file" description:"input file name" required:"true"`
 	AdminPasswd string        `long:"admin-passwd" env:"ADMIN_PASSWD" required:"true" description:"admin basic auth password"`
-	Timeout     time.Duration `long:"timeout" default:"15m" description:"remap timeout"`
+	Timeout     time.Duration `long:"timeout" default:"60m" description:"remap timeout"`
 	CommonOpts
 }
 

--- a/backend/app/cmd/restore.go
+++ b/backend/app/cmd/restore.go
@@ -12,7 +12,7 @@ type RestoreCommand struct {
 	ImportFile string `short:"f" long:"file" default:"userbackup-{{.SITE}}-{{.YYYYMMDD}}.gz" description:"file name" required:"true"`
 
 	Site        string        `short:"s" long:"site" env:"SITE" default:"remark" description:"site name"`
-	Timeout     time.Duration `long:"timeout" default:"15m" description:"import timeout"`
+	Timeout     time.Duration `long:"timeout" default:"60m" description:"import timeout"`
 	AdminPasswd string        `long:"admin-passwd" env:"ADMIN_PASSWD" required:"true" description:"admin basic auth password"`
 	CommonOpts
 }

